.github/workflows/ci.yaml

```diff
@@ -20,6 +20,25 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
 
+      - name: Set up Node
+        uses: actions/setup-node@v1
```

Comment:
In some projects I've had issues where packages are incompatible with a new version of node. So what I've tended to do is use [nvm](https://github.com/nvm-sh/nvm) to "pin" a node version in my local environment with a `.nvmrc` file. Then in GitHub Actions I use that same file to set the node version: https://github.com/lsst-sqre/www_lsst_io/blob/ead7d71212b56431288f752ed5547347f7ae9634/.github/workflows/ci.yaml#L17-L24

Not to say that the same problem will pop up here, especially if packages are being continuously kept up to date, but I think its a nice technique.

Reply:
`nvm` isn't packaged for Debian, so it's a bit painful to try to use this overall development path, but it does seem like a good idea to do GitHub Actions testing with a consistent version of Node. I'll adopt this and pin to the version of Node in Debian, I think.